README.md

```diff
@@ -1,181 +1,248 @@
 
-```
-vscode  main ❯ npx oxlint@latest src
-Finished in 388ms on 3477 files with 45 rules using 8 threads.
-Found 798 warnings.
-```
+* The umbrella crate [oxc][docs-oxc-url] exports all public crates from this repository.
+* The AST and parser crates [oxc_ast][docs-ast-url] and [oxc_parser][docs-parser-url] are ready for use.
+* See `crates/*/examples` for example usage
 
-And also in a huge monorepo using Mac i7:
+The crates are lightweight and tuned for compilation speed. This is demonstrated by our [CI runs](https://github.com/web-infra-dev/oxc/actions/workflows/ci.yml?query=branch%3Amain),
+where warm runs complete in 5 minutes.
 
-```
-Finished in 5568ms on 51931 files with 45 rules using 12 threads.
-```
+### Node.js
 
-On my Intel i7 6-core, the linter is around 80 times faster than ESLint.
+* You may use the parser via napi: [@oxidation-compiler/napi][npm-napi]
 
-See [benchmark](./benchmark/) for details.
+----
 
-### Try it out yourself!
+## 🎯 Tools
 
-The linter is currently usable and it can potentially catch a few mistakes for you:
+- [AST and Parser](#-ast-and-parser)
+- [Linter](#-linter)
+- [Resolver](#-resolver)
+- [Minifier](#-minifier)
+- [Formatter](#-formatter)
+- [Transpiler](#-transpiler)
+- [Ezno Type Checker](#-ezno-type-checker)
 
-```
-npx oxlint@latest path
-```
+### 🔸 AST and Parser
 
-## Parser Conformance
+Oxc incorporates its own AST and parser, which is by far the fastest and most conformant  JavaScript and TypeScript (including JSX and TSX) parser developed in Rust.
 
-The `cargo coverage` command reports the following conformance summary
+As the parser often represents a key performance bottleneck in JavaScript tooling,
+any minor improvements can have a cascading effect on our downstream tools.
+By developing our parser, we have the opportunity to explore and implement well-researched performance techniques.
 
+To ensure an optimal authoring experience with downstream tools,
+the AST deviates from [estree] and conforms to the ECMAScript specification.
+Notably, the AST eliminates any ambiguous nodes.
+For instance, Oxc distinguishes between `BindingIdentifier`, `IdentifierReference`, and `IdentifierName` instead of using a generic [estree] `Identifier`.
+These distinctions greatly improve the experience for developers who have previously worked with [estree].
+
+#### 🏆 Parser Performance
+
+Our [benchmark][parser-benchmark] reveals that the Oxc parser surpasses the speed of the [swc] parser by approximately 2 times and the [Rome] parser by 3 times.
+
+<details>
+  <summary>How is it so fast?</summary>
+  <ul>
+    <li>AST is allocated in a memory arena ([bumpalo]) for fast AST memory allocation and deallocation</li>
+    <li>Short strings are inlined by [CompactString]</li>
+    <li>No other heap allocations are done except the above two</li>
+    <li>Scope binding, symbol resolution and complicated syntax errors are not done in the parser, they are delegated to the semantic analyzer</li>
```

Comment:
You should define "complicated", to assess a bar

Reply:
Changing to "some syntax errors", I don't have a bar for "complicated"

Reply:
One more thing. I think you say "when" these cases are caught. It makes sense to say that they are delegated to the semantic model, but "when" does the semantic model come to play?